org-code-javabuilder/lib/src/main/java/org/code/javabuilder/UserClassLoader.java

```diff
@@ -63,25 +63,43 @@ public UserClassLoader(URL[] urls, ClassLoader parent, List<String> userProvided
       Set.of(
           "java.lang.ArithmeticException",
           "java.lang.ArrayIndexOutOfBoundsException",
+          "java.lang.Boolean",
```

Comment:
small nit, not a blocker for this PR - do you think it might be better for us to get the class names programmatically (e.g. `Boolean.class.getName()`) for these rather than typing them out manually? just to help avoid any accidental errors

Reply:
that's a good idea, I think I'll do that as a follow-up to keep this PR as just the new classes we are allowing